docs/java-rest/high-level/apis/search.asciidoc

```diff
@@ -178,3 +191,51 @@ cases need to be case accordingly:
 --------------------------------------------------
 include-tagged::{doc-tests}/SearchDocumentationIT.java[search-hits-singleHit-source]
 --------------------------------------------------
+
+===== Accessing Aggregations
+
+Aggregations can be retrieved from the `SearchResponse` by first getting the
+root of the aggregation tree, the `Aggregations` object, and then getting the
+aggregation by using its name. The following example shows how to retrieve the
```

Comment:
I would also skip the "the following example shows..." part, I would just show the code and then add a note below if there's anything to explain about it. I would do it in other places in this page too.